client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -111,7 +111,9 @@
                         <button type="button"
                             ng-click="spellcheckMenu.spellcheck()"
                             ng-disabled="spellcheckMenu.isAuto"
-                            translate>Check spelling</button>
+                            translate>
+                                <span class="shortcut pull-right">Ctrl+Shift+S</span>
```

Comment:
Should that be "translated" for Mac users somehow?

(meh, screw them :grinning: )